dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb

```diff
@@ -1,37 +1,31 @@
 class Api::V1::PeerReviewSubmissionsController < ApplicationController
   authorize_resource class: :peer_review_submissions
-  # GET /api/v1/peer_review_submissions/index_escalated
-  def index_escalated
-    # Get a list of submissions that have been escalated
-    # This will return an array of objects containing
-    # - Submitter
-    # - Course name
-    # - Unit
-    # - Level name
-    # - Submission date
-    # - Date it was escalated
-    # - ID of review
+  # All calls here will return an array of objects containing
+  # - Submitter
+  # - Course name
+  # - Unit
+  # - Level name
+  # - Submission date
+  # - Date it was escalated
+  # - ID of review
 
-    escalated_submissions = Hash.new
+  def index
+    # Get a list of submissions that are currently open
+    submissions = Hash.new
 
-    escalated_reviews = PeerReview.escalated.limit(50)
-
-    escalated_reviews.each do |review|
-      plc_course_unit = review.script.plc_course_unit
+    case params[:filter]
+      when 'escalated'
+        reviews = PeerReview.escalated.limit(50)
+      when 'open'
+        reviews = PeerReview.where(reviewer: nil).limit(50)
+      else
+        reviews = PeerReview.all.limit(50)
+    end
 
-      escalated_submissions[review.user_level.id] = {
-        submitter: review.submitter.name,
-        course_name: plc_course_unit.plc_course.name,
-        unit_name: plc_course_unit.name,
-        level_name: review.level.name,
-        submission_date: review.created_at.strftime("%-m/%-d/%Y"),
-        escalation_date: review.updated_at.strftime("%-m/%-d/%Y"),
-        review_id: review.id
-      }
+    reviews.each do |review|
+      submissions[review.user_level.id] = review.submission_summarize
```

Comment:
there are two reviews created for each user_level, right? I can see how that wasn't as much of problem before given the unlikeliness of both reviewers escalating, but it seems like a pretty big issue now.

Reply:
Discussed offline